node_modules/gh-events/lib/api.js

```diff
@@ -197,6 +197,11 @@ var getEvent = module.exports.getEvent = function(ctx, id, callback) {
             return callback({'code': 401, 'msg': 'This event cannot be retrieved on this application'});
         }
 
+        // Hide organisers their personal information
+        _.each(event.Organisers, function(organiser) {
```

Comment:
Wouldn't it be better to expose a `hideUsers` function in the users API and call that everywhere?